csfieldguide/static/interactives/scene-editor/js/scene-editor.js

```diff
@@ -902,3 +920,36 @@ function createTinyaxisMesh() {
   var object = new THREE.Mesh( geometry, material );
   return object;
 }
+
+
+/** Called on a paste event. Handles populating inputs copied from matrix simplifier 
+ *  or simply validates pasted input if it appears it hasn't been copied from
+ *  matrix simplifier.
+ */
+function populateInputsFromPaste(inputDiv) {
+  pastedData = inputDiv.val();
+  if (pastedData.includes(',v,')) {
+    // likely to be copied from matrix-simplifier
+    // Paste data into input boxes
+    eqtnData = pastedData.split(',v,'); // splits into matrix and vector values
+    matrixData = eqtnData[0].split(',');
+    vectorData = eqtnData[1].split(',');
+
+    $('#matrix-row-0-col-0').val(matrixData[0]);
+    $('#matrix-row-0-col-1').val(matrixData[1]);
+    $('#matrix-row-0-col-2').val(matrixData[2]);
+
+    $('#matrix-row-1-col-0').val(matrixData[3]);
+    $('#matrix-row-1-col-1').val(matrixData[4]);
+    $('#matrix-row-1-col-2').val(matrixData[5]);
+
+    $('#matrix-row-2-col-0').val(matrixData[6]);
+    $('#matrix-row-2-col-1').val(matrixData[7]);
+    $('#matrix-row-2-col-2').val(matrixData[8]);
```

Comment:
I did wonder.. still not sure why. 

Also with my changes since your review, the above input will actually put `1,2,3` in the vector. I don't think I can validate it perfectly anyways and is probably too much trouble for what it's worth.

Reply:
So long as we can account for both "m,x,x,x..." and "v,x,x,x" (plus the standard code) I think it's ok